ppr-ui/src/components/search/BusinessSearchAutocomplete.vue

```diff
@@ -118,7 +122,8 @@ export default defineComponent({
     }
 
     const isBusinessTypeSPGP = (businessType: BusinessTypes): boolean => {
-      return [BusinessTypes.GENERAL_PARTNERSHIP, BusinessTypes.SOLE_PROPRIETOR].includes(businessType)
+      // include all business types for PPR business searches
+      return [BusinessTypes.GENERAL_PARTNERSHIP, BusinessTypes.SOLE_PROPRIETOR].includes(businessType) && !props.isPPR
```

Comment:
Pretty minor but the flag check might make sense to do before searching the array.

Reply:
You mean to check if `businessType` is present? Because array is always there.

Reply:
Sorry, not exactly. 
Meant to verify the props flag is true before searching the the array for `businessType` 
For the sake of processing, just minor.

Reply:
It would return false in case `businessType` is null or undefined, which I think is sufficient.

Reply:
haha yea i know. 
I mean, why do the search at all if you haven't asked it too (ie the `props.isPPR`)